src/main/features/core/tray.js

```diff
@@ -6,6 +6,24 @@ import { showDesktopSettings } from './desktopSettings';
 let appIcon = null;
 const mainWindow = WindowManager.getAll('main')[0];
 
+const trayNormalPath = path.resolve(`${__dirname}/../../../assets/img/`);
+const trayPausedPath = path.resolve(`${__dirname}/../../../assets/img/paused/`);
+const trayPlayingPath = path.resolve(`${__dirname}/../../../assets/img/playing/`);
+
+const changeTrayIcon = (iconFolderPath) => {
```

Comment:
As mentioned before, can you extract this `if` logic to the top of the file like this

let appIconFileName;
```js
if (process.platform === 'darwin') {
   appIconFileName = 'macTemplate.png';
 } else if (WindowManager.getWindowManagerGDMName() === 'kde-plasma') {
   // TODO: Change this back to ico when electron supports it on linux
   // appIcon = new Tray(path.resolve(`${__dirname}/../../../assets/img/main.ico`));
   appIconFileName = 'main_tray_white_s.png';
 } else if (process.platform === 'linux') {
   appIconFileName = 'main_tray_white_s.png';
 } else {
   appIconFileName = 'main_tray_s.png';
 }
```

Then we don't need to have the `if` block twice in the same file.  For instance the logic below becomes as simple as

```js
appIcon.setImage(path.resolve(iconFolderPath, appIconFileName));
```